### PR TITLE
Update to work with ember 1.5.1, ember data 1.0.0

### PR DESCRIPTION
**Pull request by barberj**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,3 @@
 /log/*.log
 /tmp
 .DS_Store
-
-# Ignore environment specific configuration
-/config/database.yml
```
Comment (m1k3) [inline .gitignore:20]: why remove the ignore on database.yml?

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.0.0
```
Comment (m1k3) [inline .ruby-version]: this should be 2.1.2

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,5 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+Todo.create(:title => 'Get ruby 2.0.0/Rails 4 running', :is_completed => true)
```
Comment (m1k3) [inline db/seeds.rb]: we should use the new hash syntax

**Comment by m1k3** [comment]
@barberj thanks for the pull-request. I pointed out a few minor issues. When these get sorted out I will merge this in.

**Comment by barberj** [comment]
@m1k3 clearing out old pull requests. Wanna merge or should i close?

**Pull request merged**